dls-controls/malcolmjs
# Feature/upgrade material ui

## Description

Migrated everything to material-ui v1.0.0

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run `npm start` and make sure the site still works.

## Agile board tracking
